velox/docs/README.md

```diff
@@ -80,3 +76,19 @@ Below are a few monthly update guidelines.
 - Group similar contributions together. Example: All Presto scalar functions can be
   combined in one line.
 - Add a reference to the PR if the update requires more context.
+
+### Credits
+A list of contributors with their contribution count, name, and optional affiliation.
+An affiliation is the name of the company they are associated with.
+The `credits.py` program tries to automatically infer an affiliation from an email using the
+`affiliations_map.txt` file. New entries can be added to this file.
+An individual user can also add a custom affiliation in the `mailmap_base64` file.
+A user must decode the file using base64 to add an entry and then encode it again.
+Each entry is in the `.mailmap` format: `Name <Affiliation> email`.
```

Comment:
maybe add a quick piped bash example that decodes, adds a new email, then re-ecodes it, for folks not fluent in bash commands?